docs/reference/query-dsl/span-not-query.asciidoc

```diff
@@ -1,7 +1,8 @@
 [[query-dsl-span-not-query]]
 === Span Not Query
 
-Removes matches which overlap with another span query. The span not
+Removes matches which overlap with another span query. or which are
+within x tokens before or y tokens after another SpanQuery. The span not
```

Comment:
To me, having x and y mentioned without later reference is a bit confusing. If this is what pre/post is refering to, I'd either mention those parameters here (e.g. "within a certain amount of tokens (determined by the pre parameter) before") even if that gets a bit longer.